environment/providers/kubernetes/kubernetes_test.go

```diff
@@ -37,11 +37,27 @@ func runSecretInformer(ctx context.Context, clientset kubernetes.Interface) core
 	return secretInformer
 }
 
+func runCMInformer(ctx context.Context, clientset kubernetes.Interface) coreinformers.ConfigMapInformer {
```

Comment:
could you please add comments on why is this needed?

Reply:
The Kubernetes environment provider fetches the certificate from a configmap